lib/other.js

```diff
@@ -267,15 +267,25 @@ function trackDoneRequest(req, res) {
 function mount(server) {
     server.get({
         path: '/ping',
-        name: 'ping'
+        name: 'GetPing'
     }, ping);
 
     // Docs
-    server.get('/', redirect);
-    server.get(/^\/docs\/?/, redirect);
+    server.get({
+        path: '/',
+        name: 'GetDocs'
```

Comment:
Perhaps a different name for this one?  GetRoot or something?

Reply:
`GetRoot`  seems like a better name.